### PR TITLE
Add tests AfterEach nil checks

### PR DESCRIPTION
**Pull request by arnongilboa**
**What this PR does / why we need it**:
CI lanes have [lots of panicks](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/3292/pull-containerized-data-importer-e2e-nfs/1795116911005011968) in `AfterEach`.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by ShellyKa13** [comment]
/lgtm
hopefully now you got it all :) nice catch!
btw not sure what is up with the sonarcloud security hotspots

File: tests/clone-populator_test.go [hunk 1]
```diff
@@ -366,7 +366,7 @@ var _ = Describe("Clone Populator tests", func() {
 		})
 
 		AfterEach(func() {
-			if snapshot != nil {
+			if snapshot == nil {
```
Comment (akalenyu) [inline tests/clone-populator_test.go:369]: Not sure what's happening here?

**Review by akalenyu — COMMENTED** [review]
Looks good, thanks!

File: tests/clone-populator_test.go (hunk 1 above)
Comment (arnongilboa) [reply tests/clone-populator_test.go:369]: It was a buggy check. We deleted the snapshot only if it was nil...
